stregsystem/static/stregsystem/gdpr.js

```diff
@@ -6,14 +6,24 @@ Simply replaces the letters with 'x'.
 var username_element = document.querySelectorAll(".username");
 
 if (username_element.length > 0) {
-  setTimeout(function () {
-    var username_element = document.querySelectorAll(".username");
-    for (var i = 0; i < username_element.length; i++) {
-      username_element[i].innerText = "xxxxxxxx";
-    }
-    var remainingbalance_element = document.querySelectorAll(".remainingbalance");
-    for (var i = 0; i < remainingbalance_element.length; i++) {
-      remainingbalance_element[i].innerText = "xx.xx";
-    }
-  }, 5000);
+    setTimeout(function () {
+        var username_element = document.querySelectorAll(".username");
+        var replaced_username = "[brugernavn skjult]";
+        var date = new Date();
+        // Only do username replacements on Fridays
+        if (date.getDay() === 5) {
```

Comment:
```suggestion
        if (date.getMonth() === 3 && date.getDay() == 1) {
```

I think having it every friday will kill the joke, and would create a lot of confusion. Having it on april fools would make people confused but they'd be able to reason with it. Maybe extend the easter egg to the first monday after april fools to make sure it will go in effect

Reply:
Having it once a year is too short of a timeframe for it to be noticed, as most fembers doesn't stay around to look at the replaced 'xxxxxxxx'.

The first Friday of every month (for example) gives ~4 days per semester for fembers to notice it.

Reply:
I agree with Falke